lib/3scale/backend/storage.rb

```diff
@@ -182,6 +182,9 @@ def ensure_url_param(options)
           #   { host: "hostN", port: "portN" }
           # ]
           def cfg_sentinels_handler(options)
+            # get role attr and remove from options
+            # will only be validated and included when sentinels are valid
+            role = options.delete :role
```

Comment:
I think it would be better to move this to the end of the method where `role` used. There are many lines between the assignment of `role` and its usage.